src/nebula-hooks/use-announce-and-translations.ts

```diff
@@ -8,6 +8,10 @@ enum AnnouncerElements {
   SECOND = 'second-announcer-element',
 }
 
+interface Translator extends stardust.Translator {
+  language: () => string;
+}
```

Comment:
I've probably mentioned this at some point. Nebulas API for a lot of the stardust hooks is wrong. Optimally we should be able to use the type from stardust. Or am I missing a reason that it i wont be possible?

Reply:
you are right. It is wrong in nebula types so we need to extend it as you did in https://github.com/qlik-oss/sn-table/pull/611/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR22.

Reply:
Can you just use that code in this PR?

Reply:
you mean use 
```
interface ExtendedTranslator extends stardust.Translator {
  language: () => string;
}
```
right?
language: () => string; and language(): string; is same

Reply:
Ah I missed that you are extending it already, then this is fine. not sure where it should live. But that is a minor problem. I could be merged like this for now

Reply:
okay